project-reports/2023/2023-Q4-Hyperledger-Caliper.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: default
+title: 2023 Q4 Hyperledger Caliper
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+* 2 active maintainers
+* Questions on Discord and GitHub are regularly answered.
+* Monthly contributor calls if needed (discussion happens mostly on Discord).
+* No project core contributions.
+
+# Questions/Issues for the TOC
+
+Not at this time.
+
+# Releases
+
+The latest stable release is v0.5.0; we're also continuously publishing unstable releases after every merged PR (no new releases):
+
+* [npm](https://www.npmjs.com/package/@hyperledger/caliper-cli/v/0.5.0)
+* [DockerHub](https://hub.docker.com/layers/caliper/hyperledger/caliper/0.5.0/images/sha256-3a7df18ef6a17a8c851dac19241049be2375dc892643c576932f360ca9d7198c?context=explore)
+
+# Overall Activity in the Past Quarter
+
+The activity has been minimal in the last quarter due to the maintainers being otherwise engaged. There's an ongoing discussion about establishing a performance-related group (of some form), along with preparing workshop materials for Webinars in 2024. We think such publicity will do good for Caliper in terms of gaining more contributors.
```

Comment:
This status is similar to the last few reports for Caliper. Would it be worth considering moving the project to a [Dormant stage](https://toc.hyperledger.org/governing-documents/project-lifecycle.html#dormant)?

Reply:
Please can we wait a while on this, there are plans this year to try to expose Caliper to more people via webinars and there is a hope that this will encourage more contributions. Also maybe the TOC could approach the fabric and besu teams to see how interested they are in a load generator as these are the primary platforms that caliper tries to provide insight for and to try to encourage those projects to pick up responsibility for the fabric and besu connectors as they are in the best position to maitain and contribute to. I have tried in the past to reach out to Besu but didn't get anywhere. I was also the person who worked on the fabric connector, but now I am not involved with fabric so really need the fabric maintainers and contributors to take up or find someone to take up responsibility. This would really help to ensure Caliper remains current and provide a useful tool for these projects to use. If there is no interest from these teams after this year then maybe that would be a good time to move it to the dormain stage

Reply:
How about giving the project 6 months to find some new contributors/maintainers, and if no success move state to Dormant. Perhaps the risk of going Dormant will motivate some users, and then perhaps actually shifting to Dormant will motivate some users to pick it up (this has occurred in the past). I'd suggest writing this plan into the quarterly update so that everybody is on the same page.

Reply:
Thanks for adding the Dormant discussion to the report!